content/docs/2.4/scalers/solace.md

```diff
@@ -0,0 +1,104 @@
++++
+title = "Solace PubSub+"
+availability = "2.3+"
+maintainer = "Community"
+description = "Scale applications based on Solace PubSub+ Broker Queues"
+layout = "scaler"
+go_file = "solace_scaler"
++++
+
+### Trigger Specification
+This specification describes the `solace-queue` trigger that scales based on a Solace PubSub+ queues.
```

Comment:
The scaler is called Pub/Sub, but here we refer to queues, I think it's best to align the naming.

What do you think @zroubalik?

Reply:
I changed the doc title="Solace PubSub+ Queue" -- The trigger ID is still "solace-queue". Let me know if this is acceptable.
**_There is no consistency in the existing scalers. e.g.:_**
"Rabbit MQ Queue" -> rabbitmq
"ActiveMQ Artemis" -> artemis-queue
"IBM MQ" -> ibmmq
Some reference company, product, technology, and scaler object type. But none do all of those things consistently between the scaler name and the scaler identifier.

Reply:
Updated to scaler/trigger ID to "solace-event-queue" based on internal feedback. Doc title is also updated